src/app/campaign-details/campaign-details.component.html

```diff
@@ -25,6 +25,22 @@ <h3>Campaign Targets</h3>
     <p>Donations: {{ campaign.donationCount }}</p>
   </div>
 
+  <div *ngIf="campaign.budgetDetails.length != 0">
+      <h3>Budget Details</h3>
+      <div *ngFor="let detail of campaign.budgetDetails">
```

Comment:
@tbg-jasonfung for both things iterated over in this PR, they are semantically a list of similar un-ordered elements. Why don't we use the HTML list elements so the semantics are more accurate for all devices / bots and more likely to work with screen readers etc.?